performance/full_testenv/main.go

```diff
@@ -94,19 +93,15 @@ func main() {
 	}
 
 	log.Printf("creating topic %s", topic)
-	connection, err := kafka.DialContext(context.Background(), "tcp", kafkaHostPort)
-	if connection == nil || err != nil {
+	connection, err := kafka.NewManager(conf.ConnectorConfig{
+		Brokers: []string{kafkaHostPort},
+	}, 15*time.Second)
```

Comment:
I see that we have `conn.bridge.config.ConnectTimeout` so not sure if we want to change this one too